### PR TITLE
refactor/switch to non tag versions

### PR DESCRIPTION
**Pull request by RealShadowNova**
This PR switches to non-tag versions.

## Why?

This makes it possible for Renovate to update the dependency, which will make it easier for us to update our packages in the long run.

## Other Changes

- Remove the `@yarnpkg/plugin-version` plugin as it caused more issues then it resolved

### CODE REVIEW

File: packages/map/package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "prepublishOnly": "rollup-type-bundler"
   },
   "dependencies": {
-    "@joshdb/core": "next",
+    "@joshdb/core": "3.0.0-next.4613706cda66569b2d1dc99f06665eecec2505ce.0",
```
Comment (dan-online) [inline packages/map/package.json:24]: Why is this 3.0.0 and the rest 1.1.0?

File: packages/mongo/package.json [hunk 2]
```diff
@@ -18,8 +18,8 @@
     "prepublishOnly": "rollup-type-bundler"
   },
   "dependencies": {
-    "@joshdb/core": "next",
-    "@joshdb/serialize": "next",
+    "@joshdb/core": "3.0.0-next.4613706cda66569b2d1dc99f06665eecec2505ce.0",
```
Comment (dan-online) [inline packages/mongo/package.json:21]: Also 3.0.0 here

File: packages/redis/package.json [hunk 3]
```diff
@@ -20,6 +20,12 @@
     "release": "npm publish",
     "prepublishOnly": "rollup-type-bundler"
   },
+  "dependencies": {
+    "@joshdb/core": "3.0.0-next.4613706cda66569b2d1dc99f06665eecec2505ce.0",
```
Comment (dan-online) [inline packages/redis/package.json:24]: And here

File: yarn.lock [hunk 4]
```diff
@@ -980,8 +980,8 @@ __metadata:
   resolution: "@joshdb/json@workspace:packages/json"
   dependencies:
     "@favware/rollup-type-bundler": ^1.0.7
-    "@joshdb/core": next
-    "@joshdb/serialize": next
+    "@joshdb/core": 3.0.0-next.4613706cda66569b2d1dc99f06665eecec2505ce.0
```
Comment (dan-online) [inline yarn.lock:983]: And in multiple places in the yarn.lock

**Review by dan-online — CHANGES_REQUESTED** [review]
Josh at 3.0.0 and 1.1.0

File: packages/map/package.json (hunk 1 above)
Comment (RealShadowNova) [reply packages/map/package.json:24]: That's just what the version resolved to when being bumped. Nothing to be worried about.

**Review by dan-online — APPROVED** [review]
LGTM

**Pull request merged**